variables.tf

```
@@ -139,3 +113,9 @@ variable "extra_lb_http_tcp_listeners" {
   type        = list(any)
   default     = []
 }
+
+variable "tags" {
+  description = "Tags to attach to the eks cluster"
```

The tags are applied to much more than the EKS cluster (see [this description](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/c60b70fbc80606eb4ed8cf47063ac6ed0d8dd435/variables.tf#L8)). Because of that I made this suggestion:

```suggestion
  description = "Tags to apply to all resources created by the EKS Terraform module."
```

However, if this is not the behavior you intended, you might want to use the variable [`cluster_tags`](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/c60b70fbc80606eb4ed8cf47063ac6ed0d8dd435/variables.tf#L127C11-L127C23) to apply tags only to the EKS cluster.

---

this is the expected behaviour